probes/hasDangerousWorkflowScriptInjection/patch/testdata/realExample3_fixed.yaml

```diff
@@ -0,0 +1,47 @@
+name: Generate Release Image
+
+on:
+  pull_request:
+    paths:
+      - '**/CHANGELOG.md'
+  push:
+    paths:
+      - '**/CHANGELOG.md'
+
+env:
+  PULL_REQUEST_BODY: ${{ github.event.pull_request.body }} 
```

Comment:
For all the `pull_request.*` variables, I've shortened the envvar names to "PR_*"